cmd/nobled/main.go

```diff
@@ -12,7 +12,7 @@ import (
 func main() {
 	rootCmd, _ := cmd.NewRootCmd(
 		"noble",
-		"noble",
+		app.AccountAddressPrefix,
 		app.DefaultNodeHome,
 		"noble-1",
```

Comment:
Similar to my other comment, maybe we make the chain-id a constant value that is passed in here as well. Up to you

Reply:
I like it.